mpt_trie/src/trie_ops.rs

```diff
@@ -990,7 +1034,7 @@ mod tests {
     }
 
     #[test]
-    fn held_trie_cow_references_do_not_change_as_trie_changes() {
+    fn held_trie_cow_references_do_not_change_as_trie_changes() -> TrieOpResult<()> {
```

Comment:
I had no idea you could return results from test, cool! My personal though would be to just use `?` in the tests instead of `expect` & `unwrap`. Wdyt @muursh?